src/Core/Tool/Permissions/AclTrait.php

```diff
@@ -1,22 +1,24 @@
 <?php
 
 /**
- * Ushahidi Platform Permissionable interface
+ * Ushahidi Acl Trait
+ *
+ * Gives objects a method for storing an ACL instance.
  *
  * @author     Ushahidi Team <team@ushahidi.com>
- * @package    Ushahidi\Platform
+ * @package    Ushahidi\Application
  * @copyright  2014 Ushahidi
  * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
  */
 
 namespace Ushahidi\Core\Tool\Permissions;
 
-interface Permissionable
+trait AclTrait
```

Comment:
I think generally this style of Trait works well ie. a trait purely to handle repetitive DI